packages/gatsby-transformer-react-docgen/package.json

```diff
@@ -10,7 +10,7 @@
     "@babel/code-frame": "^7.5.5",
     "@babel/runtime": "^7.6.0",
     "@babel/types": "^7.6.1",
-    "ast-types": "^0.12.4",
+    "ast-types": "^0.13.2",
```

Comment:
@sidharthachatterjee @lannonbr @jquense Is there a way to get @renovate-bot to stop doing this?

Reply:
🤦🏻‍♂️ Ah, this got updated again. I suppose we could get renovate to ignore the package altogether but I think in this instance, the best approach would be to pin the package version until we are ready to upgrade.